### PR TITLE
[zh-cn]: create the translation of "Using DTMF"

### PR DESCRIPTION
**Pull request by tianyeeT**
### Related issues and pull requests
#16473

### CODE REVIEW

**Review by T34-active — COMMENTED** [review]
[针对 MDN 文档的本地化指南](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md)

File: files/zh-cn/web/api/webrtc_api/using_dtmf/index.md [hunk 1]
```diff
@@ -0,0 +1,412 @@
...
+
+- 一个 {{HTMLElement("audio")}} 元素用于播放由被“呼叫”的 `RTCPeerConnection` 接收到的音频。
+- 一个 {{HTMLElement("button")}} 元素用于触发创建和连接两个 `RTCPeerConnection` 对象，然后发送 DTMF 音调。
+- 一个 {{HTMLElement("div")}} 用于接收和显示日志文本，以显示状态信息。
+
+```html
+<p>
+  This example demonstrates the use of DTMF in WebRTC. Note that this example is
+  "cheating" by generating both peers in one code stream, rather than having
+  each be a truly separate entity.
```
Comment (tianyeeT) [inline files/zh-cn/web/api/webrtc_api/using_dtmf/index.md:50]:
```suggestion
  这个示例演示了在 WebRTC 中使用 DTMF。请注意，这个示例是“作弊”的，它通过在一个代码流中生成两个对等端，而不是让每个对等端都是一个真正独立的实体。
```

File: files/zh-cn/web/api/webrtc_api/using_dtmf/index.md [hunk 2]
```diff
@@ -0,0 +1,412 @@
...
+- `dtmfSender`
+  - ：连接的 {{domxref("RTCDTMFSender")}} 对象。这将在设置连接时获取，在[将音频添加到连接](#将音频添加到连接) 中的 `gotStream()` 函数中进行。
+- `hasAddTrack`
+  - ：由于一些浏览器尚未实现 {{domxref("RTCPeerConnection.addTrack()")}}，因此需要使用废弃的 {{domxref("RTCPeerConnection.addStream", "addStream()")}}方法，我们使用此布尔值来确定用户代理是否支持 `addTrack()`；如果不支持，我们将退回到 `addStream()`。这将在 `connectAndDial()` 中确定，如下[启动连接过程](#启动连接过程)所示。
+- `mediaConstraints`
+  - ：指定启动连接时要使用的约束的对象。我们只想要音频连接，所以 `video` 是 `false`，而 `audio` 是 `true`。
+- `offerOptions`
+  - ：提供选项以在调用 {{domxref("RTCPeerConnection.createOffer()")}} 时指定。在这种情况下，我们声明我们想要接收音频但不要视频。
+- `dialButton` 和 `logElement`
+  - ：这些变量将用于存储对拨号按钮和将记录信息写入的 {{HTMLElement("div")}} 的引用。它们将在页面首次加载时设置。请参见下面的 [初始化](#初始化)。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/webrtc_api/using_dtmf/index.md:105]:
```suggestion
  - ：这些变量将用于存储对拨号按钮和将记录信息写入的 {{HTMLElement("div")}} 的引用。它们将在页面首次加载时设置。请参见下面的[初始化](#初始化)。
```

File: files/zh-cn/web/api/webrtc_api/using_dtmf/index.md [hunk 3]
```diff
@@ -0,0 +1,412 @@
...
+它们依次是：
+
+- `dialString`
+  - ：当用户点击“拨号”按钮时，呼叫方将发送的 DTMF 字符串。
+- `callerPC` 和 `receiverPC`
+  - ：分别表示呼叫方和接收方的 {{domxref("RTCPeerConnection")}} 对象。这些对象将在呼叫启动时初始化，在我们的 `connectAndDial()` 函数中，如下[启动连接过程](#启动连接过程)所示。
+- `dtmfSender`
+  - ：连接的 {{domxref("RTCDTMFSender")}} 对象。这将在设置连接时获取，在[将音频添加到连接](#将音频添加到连接) 中的 `gotStream()` 函数中进行。
+- `hasAddTrack`
+  - ：由于一些浏览器尚未实现 {{domxref("RTCPeerConnection.addTrack()")}}，因此需要使用废弃的 {{domxref("RTCPeerConnection.addStream", "addStream()")}}方法，我们使用此布尔值来确定用户代理是否支持 `addTrack()`；如果不支持，我们将退回到 `addStream()`。这将在 `connectAndDial()` 中确定，如下[启动连接过程](#启动连接过程)所示。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/webrtc_api/using_dtmf/index.md:99]:
```suggestion
  - ：由于一些浏览器尚未实现 {{domxref("RTCPeerConnection.addTrack()")}}，因此需要使用废弃的 {{domxref("RTCPeerConnection.addStream", "addStream()")}} 方法，我们使用此布尔值来确定用户代理是否支持 `addTrack()`；如果不支持，我们将退回到 `addStream()`。这将在 `connectAndDial()` 中确定，如下[启动连接过程](#启动连接过程)所示。
```

File: files/zh-cn/web/api/webrtc_api/using_dtmf/index.md [hunk 4]
```diff
@@ -0,0 +1,406 @@
+---
+title: 在 WebRTC 中使用双音多频
```
Comment (yin1999) [inline files/zh-cn/web/api/webrtc_api/using_dtmf/index.md:2]:
```suggestion
title: 在 WebRTC 中使用 DTMF
```
```suggestion
title: 在 WebRTC 中使用双音多频
```

**Review by yin1999 — APPROVED** [review]
Thank you @tianyeeT

**Pull request merged**